packages/accordion/src/vaadin-accordion-heading.js

```diff
@@ -69,6 +69,7 @@ class AccordionHeading extends ActiveMixin(DirMixin(ThemableMixin(PolymerElement
         button {
           display: flex;
           align-items: center;
+          justify-content: inherit;
```

Comment:
The `justify-content: space-between` is applied to the host:

https://github.com/vaadin/web-components/blob/b16f960e73943529c60dd327d0a9a5075cf113e2/packages/details/theme/lumo/vaadin-details-summary-styles.js#L98-L105

It works for `details` but not for `accordion-panel` whose content is wrapped in a `button`.